exonum-java-binding-common/src/main/java/com/exonum/binding/common/serialization/StandardSerializers.java

```diff
@@ -48,13 +48,27 @@
     return Fixed32Serializer.INSTANCE;
   }
 
+  /**
+   * Returns a serializer of integers using variable length encoding in little-endian byte order.
+   */
+  public static Serializer<Integer> varInt32() {
```

Comment:
I’d use the naming from protobuf unless we support the only type — is it sint32/int32/uint32?

Reply:
protobuf has 3 var-ints:
- `int32` -> from the doc: _Inefficient for encoding negative numbers – if your field is likely to have negative values, use sint32 instead._
- `uint32` -> stored as signed int in Java
- `sint32` -> uses uint32 encoding with `ZigZag32` encoding.

So I believe we need serializer for `sint32` only. That's because I gave `varInt32` name. And the same story for 64-bit ints.
@dmitry-timofeev @DENMROOT @MakarovS WDYT?

Reply:
`uint32` is _represented_ as a signed int in Java, but I believe is encoded more efficiently than `sint32` because the latter performs zig-zag encoding (i.e., 127 (2^7 - 1) as `uint32` takes 1 byte; whilst the same as `sint32` takes 2 bytes): https://developers.google.com/protocol-buffers/docs/encoding#varints